### PR TITLE
Commands return a CommandResponseObject

### PR DESCRIPTION
**Pull request by o-reo**
Edited every command to return a CommandResponse object

### CODE REVIEW

File: tests/utils/test_response.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    with ctx:
+        response = CommandResponse.fail()
+        assert response.status_code == CommandResponse.STATUS_ERROR
+
+
+def test_response_dict():
+    """Testing command responses"""
+    ctx = click.Context(click.Command('cmd'))
+    with ctx:
+        response = CommandResponse(status_code=CommandResponse.STATUS_OK, data={"oups": True})
```
Comment (neomatamune) [inline tests/utils/test_response.py:25]: Why do we need an existing content for `data` if we don't check it in the following test ?

Comment (o-reo) [reply tests/utils/test_response.py:25]: Fixed

**Review by neomatamune — CHANGES_REQUESTED** [review]

File: Babylon/groups/azure/groups/arm/commands/run.py [hunk 2]
```diff
@@ -63,10 +63,10 @@ def run(
         )
     except HttpResponseError as _e:
         logger.error(f"An error occurred : {_e.message}")
-        return CommandResponse(status_code=CommandResponse.STATUS_ERROR)
+        return CommandResponse.f(status_code=CommandResponse.STATUS_ERROR)
```
[the remark this answers is not in the document]
Comment (o-reo) [reply Babylon/groups/azure/groups/arm/commands/run.py:66]: Fixed !

[the remark this answers is not in the document]
Comment (neomatamune) [reply Babylon/groups/azure/groups/arm/commands/run.py:66]: please push commit :)

**Review by neomatamune — APPROVED** [review]

**Pull request merged**